subsys/net/ip/net_core.c

```diff
@@ -65,12 +66,47 @@ void net_analyze_stack(const char *name, const char *stack, size_t size)
 }
 #endif /* CONFIG_INIT_STACKS */
 
+#if defined(CONFIG_NET_SOCKETS_RAW)
+static inline enum net_verdict process_raw_data(struct net_pkt *pkt)
+{
+	struct net_context *ctx;
+
+	ctx = net_context_find(AF_PACKET, SOCK_RAW, ETH_P_ALL);
+	if (!ctx) {
+		return NET_DROP;
+	}
+
+	/* Currently we are skipping L2 layer verification and not
```

Comment:
This comment is golden, and should go into the PR description and into commit message.

Reply:
Tried to update as many places I can in new version.